src/main/java/seedu/address/logic/commands/AppendLogCommand.java

```diff
@@ -51,7 +52,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         for (Person person : model.getFoundPersonsList()) {
             if (model.getLogBook().hasPerson(person)) {
-                continue;
+                return new CommandResult(MESSAGE_DUPLICATE);
```

Comment:
Can I ask how you wish to handle this exceptional case? This may be too naive a fix to this exceptional case in a few ways.
1. If we wish to handle the duplicate case via an error message, it is better to throw an `CommandException` rather than return a `CommandResult`, as the `CommandResult` denotes our happy path and shouldn't be misused in the error case.
     a. On this note, based on the `return` statement, it is supposed to cut the execution of this `Command` short. In this case, this should not be packaged in the same loop as `logbook.addPerson` below, as even though the error message shows, non-duplicate entries are still added up until this point.
2. If we wish to handle the duplicate clause as an additional message that can show up occasionally, but still allow the user to perform the operation as usual, we should not `return` or `throw` a message in the loop, as the execution should go to completion.
```
boolean hasDupes = false; // tells the program if the execution encountered dupes
String duplicateClause = ""; // string collection of all dupes encountered
for (Person person : model.getFoundPersonsList()) {
    if (model.getLogBook().hasPerson(person)) {
        hasDupes = true;
        duplicateClause += " " + ...some duplicate person info here...
        continue; // skip over the addition of the current person
    }
    model.getLogBook().addPerson(person);
}

return hasDupes
    ? new CommandResult(...duplicate formatted string...)
    : new CommandResult(MESSAGE_SUCCESS);
```

Reply:
Thanks for your input! I see how my implementation does not allow for any new persons to be added as long as there is at least one duplicate person present, which is not desired. I've made the relevant changes in a recent commit.